MODULE.bazel

```diff
@@ -24,6 +24,10 @@ use_repo(
 
 register_toolchains("//kotlin/internal:default_toolchain")
 
+# TODO(bencodes) We should be able to remove this once rules_android has rolled out official Bzlmod support
+remote_android_extensions = use_extension("@bazel_tools//tools/android:android_extensions.bzl", "remote_android_tools_extensions")
```

Comment:
Mirroring what's being done in `rules_android` https://github.com/bazelbuild/rules_android/blob/main/MODULE.bazel#L63-L64